modules/manage/pages/security/encryption.adoc

```diff
@@ -373,7 +392,7 @@ See also: xref:./listener-configuration.adoc[Configure Listeners]
 
 If you're using `rpk` to interact with the Kafka API using mTLS identity (for example, to manage topics or messages), pass the `--tls-key`, `--tls-cert`, and `--tls-truststore` flags to authenticate.
 
-To interact with the Admin API (for example, to manage users), pass the `--admin-api-tls-key`, `--admin-api-tls-cert`, and `--admin-api-tls-truststore` flags.
+To interact with the Admin API (for example, to manage users), pass the `--admin-api-tls-key`, `--admin-api-tls-cert`, `--admin-api-tls-truststore` flags.
```

Comment:
nit: It looks to me like the code example that follows is for Kafka API (since it uses the `--tls-` flags that aren't prefixed with `--admin-api` -- for me it might be a little less confusing if this paragraph came after the example, perhaps after "Replace placeholders"

Looks good otherwise so it's up to you!

Reply:
@kbatuigas, I've updated as requested (I hope). Could you take a quick look and confirm?

Reply:
@asimms41 Actually, now that I look at the section again, I've either made it more confusing with my suggested placement, or perhaps we don't quite need the `rpk topic describe` example later. I believe we're trying to demonstrate how to use TLS using two rpk use cases (Kafka API / managing topics, and Admin API / managing users) but now it's not so clear which code example belongs to which use case. Correct me if I've misunderstood. I think we could either: 
a) move the Admin API paragraph close to the end of the section, above "By default, `rpk` connects to `localhost`...", or 
b) remove "To check the configuration of the topic" and the accompanying code block, and in the first paragraph maybe link out to the `rpk topic create` reference doc (`For example, to create a new topic using the xref:reference/rpk/rpk-topic/rpk-topic-create.adoc[`rpk topic create`] command: `) if we want to provide more guidance for `rpk topic`.

Reply:
@kbatuigas, I've made which some changes, which are closer to (a). What do you think? I also think I'm going to raise a separate ticket to rework this section as tabbed content.

Reply:
@asimms41 looks good, thanks!